docs/databases/swift-kuery.html

```diff
@@ -252,237 +229,150 @@ <h2 class="heading-2"><span class="blue-text">Step 5 (Optional):</span> Test sav
+    // Write query and execute it here
+}</code></pre>
+      <p>Now we can build our SELECT query that will query the database for every entry in the "BookTable":</p>
+      <pre><code class="language-swift">let selectQuery = Select(from: App.bookTable)</code></pre>
       <p>Like before we can now execute the query using the `execute` method in Connection:</p>
-      <pre><code class="language-swift">func getTitles(_ callback: @escaping (String) -> Void ) -> Void {
-    pool.getConnection() { connection, error in
-      guard let connection = connection else {
-        guard let error = error else {
-          Log.error("Error connecting: Unknown Error")
-          return
-        }
-        Log.error("Error connecting: \(error)")
-        return
-      }
-
-      let selectQuery = Select(from: titles)
-
-      connection.execute(query: selectQuery) { selectResult in
-        guard let resultSet = selectResult.asResultSet else {
-          guard let error = selectResult.asError else {
-            return callback("Error executing query: Unknown Error")
-          }
-          return callback("Error executing query: \(error)")
-        }
-      }
+      <pre><code class="language-swift">connection.execute(query: selectQuery) { selectResult in
+    guard let resultSet = selectResult.asResultSet else {
+        Log.error("Error connecting: \(selectResult.asError?.localizedDescription ?? "Unknown Error")")
+        return completion(nil, .internalServerError)
     }
-  }</code></pre>
+    // Iterate through result set here
+}</code></pre>
       <p>Here we check to see if the `selectResult` value is a valid Result set otherwise we log and error and return.</p>
-      <p>Next we need to create a variable that will store the value we respond to the incoming GET request with:</p>
-      <pre><code class="language-swift">func getTitles(_ callback: @escaping (String) -> Void ) -> Void {
-    pool.getConnection() { connection, error in
-      guard let connection = connection else {
-        guard let error = error else {
-          Log.error("Error connecting: Unknown Error")
-          return
-        }
-        Log.error("Error connecting: \(error)")
-        return
-      }
-
-      let selectQuery = Select(from: titles)
-
-      connection.execute(query: selectQuery) { selectResult in
-        guard let resultSet = selectResult.asResultSet else {
-          guard let error = selectResult.asError else {
-            return callback("Error executing query: Unknown Error")
-          }
-          return callback("Error executing query: \(error)")
-        }
-        var resultString = ""
-
-      }
+      <p>Next we need iterate through our returned rows and convert them to books:</p>
+      <pre><code class="language-swift">var books = [Book]()
+resultSet.forEach() { row, error in
+    // Handle callback here
+}</code></pre>
+      <p>The `forEach` function will return either a `row`, an `error` or `nil` (Meaning there are no more rows).</p>
```

Comment:
Can we change the text in brackets to
"(in this case `nil` means that there are no more rows)."